funds-manager/src/indexer/queries.rs

```diff
@@ -56,6 +56,9 @@ pub(crate) struct FeeValue {
     /// The mint of the fee
     #[sql_type = "Text"]
     pub mint: String,
+    /// The receiver of the mint
+    #[sql_type = "Text"]
+    pub receiver: String,
```

Comment:
does this require a migration? not seeing a new one in this PR

Reply:
no, this is a query-specific struct, so it defines the return type expected by a specific query. you can see down below that i added the `receiver` field to the `SELECT` statement